source/assets/scripts/main.js

```diff
@@ -95,6 +95,7 @@ function taskCount() {
 
```

Comment:
Description to taskCount() function is too vague now. Except counting number of task done and total task for each weekday list, it also works for progress bar and display different message for different progress user have.